.circleci/workflows.yml

```diff
@@ -292,7 +292,7 @@ jobs:
           depth: 1
       - run:
           name: Generate build metadata
-          command: ferrocene/ci/scripts/generate-build-metadata.sh
+          command: ./x.py --stage 2 dist ferrocene-build-metadata
```

Comment:
Why `--stage 2`?

Reply:
I copied what the other ci steps did when invoking dist, that's why

Reply:
Oh ok! The others use `--stage 2` to make sure they don't use the stage1 compiler, but it's not really needed for this specific dist step.